pgjdbc/src/main/java/org/postgresql/core/FixedLengthOutputStream.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017, PostgreSQL Global Development Group
```

Comment:
change to 2019

Reply:
K, I'll rather change it to 2020 directly, okay?